components/__tests__/search-form.test.tsx

```diff
@@ -8,61 +8,15 @@ describe("SearchForm", () => {
   const handleSubmitKeyword = jest.fn();
 
   it("renders correctly", () => {
-    const { container } = render(
+    render(
       <SearchForm
         checkDocSize={true}
         itemName="provinsi"
         onSubmitKeywords={handleSubmitKeyword}
       />,
     );
 
-    expect(container.firstChild).toMatchInlineSnapshot(`
-      <form
-        class="pb-8 space-y-4"
-      >
-        <div
-          class="flex flex-row items-end"
-        >
-          <div
-            class="flex flex-1 items-center mt-1"
-          >
-            <div
-              class="space-y-1 flex-1"
-            >
-              <label
-                class="block text-sm font-medium text-gray-700"
-                for="keywordsInput"
-              >
-                Cari 
-                provinsi
-                :
-              </label>
-              <div
-                class="flex rounded-md shadow-sm w-full"
-              >
-                <input
-                  autocomplete="off"
-                  class="first:rounded-l-md last:rounded-r-md focus:ring-blue-500 focus:border-blue-500 block w-full text-sm border-gray-300 focus:z-10"
-                  id="keywordsInput"
-                  type="text"
-                  value=""
-                />
-              </div>
-            </div>
-          </div>
-          <div
-            class="flex flex-row mt-0 ml-2"
-          >
-            <button
-              class="flex flex-row px-4 py-2 text-sm rounded-md items-center justify-center border border-transparent font-medium shadow-sm focus:outline-none focus:ring-2 focus:ring-offset-2 text-white bg-blue-600 hover:bg-blue-700 focus:ring-blue-500 disabled:cursor-not-allowed disabled:opacity-75 flex-1"
-              type="submit"
-            >
-              Cari
-            </button>
-          </div>
-        </div>
-      </form>
-    `);
+    expect(screen.getByText(/cari provinsi/gi)).toBeInTheDocument();
```

Comment:
Please use the queries recommended in testing-playground.com, you can use its browser extension to get the proper query.

After getting the element, please try to interact with the element using userEvent.
For more information, please read through our testing guidelines in this issue #731.